AP-protocol.md

```diff
@@ -41,7 +41,7 @@ connected_drone_ids = ["connected_id1", "..."] # max 2 entries
 ```
 - note that `connected_drone_ids` cannot contain `client_id` nor repetitions
 - note that a client cannot connect to other clients or servers
-- note that a client can be connected to at most two drones
+- note that a client can be connected to at least one drone and at most two drones
```

Comment:
This may be better in another PR

Reply:
That's fine but we should move. If we pile up PR it will be kinda of a mess to merge everything on monday.